blueprint/core/blueprint_CanvasView.h

```diff
@@ -394,6 +394,45 @@ namespace blueprint
                         return juce::var();
                     }
             });
+
+            // drawImage support
+            setProperty("drawImage", juce::var::NativeFunction {
+                    [=](const juce::var::NativeFunctionArgs& args) -> juce::var {
+                        jassert(graphics);
+                        jassert(args.numArguments >= 3 && args.numArguments <= 5);
+
+                        const juce::String svg  = args.arguments[0].toString();
+                        const float        xPos = args.arguments[1];
+                        const float        yPos = args.arguments[2];
+
+                        //TODO: Add support for drawimage source width and source height to draw sub rect of an image.
+                        //      ctx.drawImage(image, sx, sy, sWidth, sHeight, dx, dy, dWidth, dHeight);
+
+                        std::unique_ptr<juce::XmlElement> svgElement  = juce::XmlDocument::parse(svg);
+                        jassert(svgElement);
```

Comment:
Here I think we can be slightly more forgiving– we have the `if (svgElement)` below and the `jassert` here. I think we can just nudge that to:

```
if (!svgElement) {
    DBG("WARNING: Invalid SVG string supplied to `drawImage`.");
    return juce::var();
}
```

I think favoring warnings + noops when we can just helps out the hot reload stuff. It'd be marginally annoying if I had to recompile just because I put a typo in my javascript that calls `drawImage` :)

Reply:
Yeah cool makes sense! Will get that sorted now.

Reply:
@nick-thompson, tidied up in https://github.com/nick-thompson/blueprint/pull/57/commits/d52953cedb899331b8059ee178e6c85493f89221